tortoise/util.go

```diff
@@ -53,7 +73,7 @@ func (a vec) String() string {
 
 func calculateOpinionWithThreshold(logger log.Log, v vec, layerSize int, theta uint8, delta float64) vec {
 	threshold := float64(theta) / 100 * delta * float64(layerSize)
-	netVote := float64(v[0] - v[1])
+	netVote := float64(v.netVote())
```

Comment:
How is it ok that we use floats here? I thought that they are unacceptable in consensus code due to varying implementation / behaviour across processor architectures 🤔

Reply:
Let's continue the conversation in #2629